src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -339,4 +357,59 @@ private function announce($subject, Transition $initialTransition, Marking $mark
             $this->dispatcher->dispatch(sprintf('workflow.%s.announce.%s', $this->name, $transition->getName()), $event);
         }
     }
+
+    /**
+     * @param string $transitionName
+     *
+     * @return Transition[]
+     */
+    private function getTransitionsByName(string $transitionName): array
+    {
+        $transitions = array_filter(
```

Comment:
you can return directly instead of using a var

Reply:
Yeah, I know. I just have this habit of not immediately returning something, that I might need to xdebug, so I can set a breakpoint on the local variable and see the result of the complex instruction easier.

I changed the code, as you asked.